scikit_wrappers.py

```diff
@@ -321,7 +324,7 @@ def encode(self, X, batch_size=50):
                         batch = batch.cuda(self.gpu)
                     features[
                         count * batch_size: (count + 1) * batch_size
-                    ] = self.encoder(batch)
+                    ] = self.encoder(batch).cpu()
```

Comment:
Could you add the same .cpu() call on line 338 in case all time series do not have equal lengths?